net-im/swiften/swiften-4.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit scons-utils toolchain-funcs
+
+DESCRIPTION="A C++ library for implementing XMPP applications"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/swift-${PV}/swift-${PV}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc expat icu idn sqlite upnp zeroconf"
+
+DEPEND="dev-libs/boost:=
+	dev-libs/openssl:0=
+	sys-libs/zlib
+	doc? ( app-text/docbook-xsl-stylesheets
+		app-text/docbook-xml-dtd:4.5 )
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2 )
+	icu? ( dev-libs/icu:= )
+	idn? ( net-dns/libidn )
+	sqlite? ( dev-db/sqlite:3 )
+	upnp? (	net-libs/libnatpmp
+		net-libs/miniupnpc )
```

Comment:
`:=`. Please check for other `:=` candidates too.

Reply:
So, if a package has also a sub slot, I should do this, correct?

Reply:
Hope, this are all now :/